packages/core/src/MapIt/index.js

```diff
@@ -358,6 +358,11 @@ function MapIt({
             });
             setGeoStyles(layer, geoColor, "blur");
           }
+          let label = feature.properties.name;
+          if (vulnerabilityLabel && feature.properties.score) {
+            label = `${label} <br /> ${vulnerabilityLabel} : ${feature.properties.score}`;
+          }
+          layer.bindTooltip(label, { direction: "auto" });
```

Comment:
Does the tooltip support styles like bold, etc.? It will be cool to make say the score standout.